### ISSUE TITLE
Add an optional notification when players switch from private to non-private server and vice-versa

### ISSUE THREAD

**Issue by arskvsh** [NewAmazingPVP/ProxyPlayerNotify#5]
Hello! Thank you for making a neat plugin, it seems that there is no a better alternative by far. Though, I have a proposal to tweak its functionality a bit.

The idea is to implement a possibility of enabling a notification when player switches from a private server to a non-private one. In that case the plugin should send a join message instead of a switch message, like if a player have just joined the network. The reverse action can also be implemented, so everybody thinks that a player left the network, while they just went to a private server.

This also could be considered for implementing as default behaviour cause it excludes situations when somebody moves between public and private part of the network stealthily. And the current behaviour could be possibly reworked as part of vanishing functionality.

**Example:** I have a limbo server, which, of course, works as a pass-through most of the time. So, when a player joins limbo, the whole network gets a notification. Then a player switches from limbo to a game server, notifying the whole network several seconds after. That behaviour seems to be quite annoying. Though, If I make the limbo private in PPN config, players just slip to all game servers silently.

**Comment by NewAmazingPVP** [comment]
Hello @arskvsh, I've decided to introduce a new configuration option for limbo servers, as I didn't want to alter the behavior of private servers, as some users preferred it that way. Please test this [build](https://cdn.discordapp.com/attachments/1103725182314614834/1264621003884072970/ProxyPlayerNotify-limboServers.jar?ex=669e8996&is=669d3816&hm=ded6c82f91afb09d9c303731290d49c7fa132ca5c323eda4142bdefb0bff76cf&) out and let me know if it works as intended.

Please regenerate the config.yml and here is the newly implemented option.
```yml
# Limbo Servers
# Specify the limbo servers (lowercase) where player join, leave, and switch notifications should be managed differently.
# These servers act as pass-throughs most of the time and can be configured to adjust notification behavior accordingly.
# When a player joins a limbo server, no network-wide join notification is sent.
# When a player switches from a limbo server to a game server, it should send a join notification as if the player is joining the network for the first time.
# Conversely, when a player switches from a game server to a limbo server, it should send a leave notification as if the player is leaving the network.
# This configuration helps avoid unnecessary notifications and prevents stealthy movements between public and private parts of the network.
# In short: Join and leave notifications are sent based on transitions to and from these servers to manage network-wide notifications effectively.
LimboServers:
  - "limbo-afk"

**Comment by arskvsh** [comment]
> Please test this [build](https://cdn.discordapp.com/attachments/1103725182314614834/1264621003884072970/ProxyPlayerNotify-limboServers.jar?ex=669e8996&is=669d3816&hm=ded6c82f91afb09d9c303731290d49c7fa132ca5c323eda4142bdefb0bff76cf&) out and let me know if it works as intended.

Hello, thank you for a quick response! I've just briefly tested the feature and it seems to be functioning perfectly. No matter if it's default behaviour or not, a working solution for the key issue is now provided. Thank you again!

**Comment by NewAmazingPVP** [comment]
That's great to hear! This feature should be released with the new version. Please reopen if there are any issues.
